pandas/tests/groupby/test_groupby.py

```diff
@@ -2174,3 +2174,14 @@ def test_groupby_series_with_tuple_name():
     expected = Series([2, 4], index=[1, 2], name=("a", "a"))
     expected.index.name = ("b", "b")
     tm.assert_series_equal(result, expected)
+
+
+def test_groupby_std_on_nullable_column():
```

Comment:
can you make this more general and test var at least as well

Reply:
I added var to the tests, but I don't know how to make them more general, if you tell me how this can be achieved I will be happy to solve it